### PR TITLE
Add profile router.Add profile handlers,requests,responses. Add tests.

### PR DESCRIPTION
**Pull request by Kun-Kun**
Added router, for now without a factory, but I will implement it after merge all rourers to master branch.
Large branch because a lot of requests, responses and handlers in this router.

### CODE REVIEW

File: softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Contact.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.softgroup.common.protocol.entry;
+
+import java.util.ArrayList;
+
+/**
+ * Created by user on 26.02.2017.
+ */
+public class Contact {
+    //номера телефонов
```
Comment (Dominys) [inline softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Contact.java]: @Kun-Kun English!

File: softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Profile.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
+package com.softgroup.common.protocol.entry;
+
+/**
+ * Created by user on 26.02.2017.
+ */
+public class Profile {
+    //ключ
```
Comment (Dominys) [inline softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Profile.java]: @Kun-Kun English!

**Comment by Dominys** [comment]
@Kun-Kun Entry must be named DTO and all classes in that package must have prefix DTO

**Comment by Dominys** [comment]
@Kun-Kun And one more 44 files... Are you kidding??

**Comment by Kun-Kun** [comment]
i'm not kidding, but my task is adding this router, and it will be not completely if i separate it into different branch.Every Handler use different request and response class,every request and response use DTO classes. Therefore this branch so large

File: softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Contact.java (hunk 1 above)
Comment (Kun-Kun) [inline softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Contact.java]: Ok, this code and comments generated from API

File: softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Profile.java (hunk 2 above)
Comment (Kun-Kun) [inline softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/entry/Profile.java]: Ok, this code and comments generated from API

**Comment by Kun-Kun** [comment]
> Entry must be named DTO and all classes in that package must have prefix DTO
Some classes (in Models) already have DTO postfix. Are your sure that name  DTOMessageRequestDTO good?
In message branch my entry folder have internal DTO folder, that consist all classes with postfix DTO, named according to Models.xls.

**Comment by Dominys** [comment]
@Kun-Kun This PR contains same code as this one https://github.com/Kun-Kun/Server/pull/2 
Close duplicate PRs.

**Pull request merged**